manifests/h/HiroshibaKazuyuki/VOICEVOX/CPU/0.14.10/HiroshibaKazuyuki.VOICEVOX.CPU.locale.en-US.yaml

```diff
@@ -9,8 +9,8 @@ PublisherUrl: https://voicevox.hiroshiba.jp
 PublisherSupportUrl: https://github.com/VOICEVOX/voicevox/issues
 PackageName: VOICEVOX
 PackageUrl: https://github.com/VOICEVOX/voicevox
-License: LGPL-3.0 license
-LicenseUrl: https://github.com/VOICEVOX/voicevox/blob/main/LICENSE
+License: Proprietary
+LicenseUrl: https://raw.githubusercontent.com/VOICEVOX/voicevox_blog/4a33d321d292071152633a650a59006b21d16a61/src/markdowns/softwareReadme.md

```

Comment:
Why use the license from `VOICEVOX/voicevox_blog` when the installer is at `VOICEVOX/voicevox`

Reply:
The releases we provide consist of several components hosted in different repositories.

- https://github.com/VOICEVOX/voicevox (Electron application dual-licensed under `LGPL-3.0` `OR` a "Proprietary" license)
- https://github.com/VOICEVOX/voicevox_engine (local FastAPI server that `voicevox` uses)
- https://github.com/VOICEVOX/voicevox_core (DLL that `voicevox_engine` uses)
- https://github.com/VOICEVOX/voicevox_resource (proprietary resources that the three mentioned above use)

The EULA that VOICEVOX reproduces and that users must agree to before using is not in github.com/VOICEVOX/voicevox. It is [embedded on GHA](https://github.com/VOICEVOX/voicevox/blob/0.14.10/.github/workflows/build.yml#L184-L190). The GFM file on voicevox_blog is for <https://voicevox.hiroshiba.jp/term/>, but is same content. `LICENSE` in github.com/VOICEVOX/voicevox just says the repository is dual-licensed, the first license is LGPL-3.0, and the second one is "all rights reserved".

However, I changed `LicenseUrl` in another reason.
[`e1deee1` (#133789)](https://github.com/microsoft/winget-pkgs/pull/133789/commits/e1deee1eb13607bfbcd929d582620351e943d302)

We discussed on our side and realized `LicenseUrl` should not be permanent in our case. When we change the EULA, it is applied for past distributions. So I changed it to <https://voicevox.hiroshiba.jp/term/>.

@Hiroshiba Right?

Reply:
That's correct. When you launch the app, the Terms of Use are displayed initially.